.github/workflows/python-publish.yml

```diff
@@ -13,7 +13,8 @@ on:
     types: [published]
 
 permissions:
-  contents: read
+  contents: write
```

Comment:
is this permission needed to run the `gh release upload` commands?

Reply:
yes, that is required for that command